config/zones.json

```diff
@@ -120,12 +120,13 @@
   },
   "BE": {
     "capacity": {
-      "biomass": 1078,
-      "coal": 0,
-      "gas": 5303,
-      "hydro": 1425,
+      "biomass": 1076,
+      "coal": 0, // ENSTSOE 470 figure outdated
```

Comment:
You need to remove all comments (not json compliant)

Reply:
can I store them somewhere else in the code? we'll need them later